src/DEFCON-1.sol

```diff
@@ -15,6 +15,8 @@
 
 pragma solidity 0.5.12;
 
+import "lib/dss-interfaces/src/dss/ChainlogAbstract.sol";
```

Comment:
Rather than importing this line in these files we can probably just stub the interface. That way we simplify the flattening process for the actual DEFCON spells. The interface imports are fine in the tests, but maybe replace this line with:

```
contract ChainlogAbstract {
	function getAddress(bytes32) external view returns (address);
}
```